app/src/main/java/com/mertceyhan/bitcoinmarket/components/Chart.kt

```diff
@@ -41,6 +44,12 @@ fun Chart(
         modifier = modifier
             .fillMaxWidth()
             .requiredHeight(300.dp)
+            .placeholder(
+                visible = isShowShimmer,
+                color = ShimmerProperty.color,
+                shape = ShimmerProperty.shape,
+                highlight = ShimmerProperty.highlight
+            )
```

Comment:
I think this is too much duplication, maybe we can create our own extension for reducing it.

```
package com.mertceyhan.bitcoinmarket.utils.extensions

import androidx.compose.ui.Modifier
import com.google.accompanist.placeholder.material.placeholder

fun Modifier.placeholder(visible: Boolean) = placeholder(
    visible = visible,
    color = PlaceHolderPromerty.color,
    shape = PlaceHolderPromerty.shape,
    highlight = PlaceHolderPromerty.highlight
)
```